src/hiero_sdk_python/query/transaction_record_query.py

```
@@ -58,16 +61,29 @@ def _make_request(self):
             if not hasattr(query, 'transactionGetRecord'):
                 raise AttributeError("Query object has no attribute 'transactionGetRecord'")
             query.transactionGetRecord.CopyFrom(transaction_get_record)
-            
+    
             return query
         except Exception as e:
             print(f"Exception in _make_request: {e}")
             raise
+    def _map_record_list(
+        self,
+        proto_records: List[transaction_record_pb2.TransactionRecord]
```

Ruff (0.14.14)

[error] Undefined name `List`

(F821)